### DISCUSSION TITLE
Live painting doesn't work with selection

### DISCUSSION THREAD

**Discussion by brightcxy** [Acly/krita-ai-diffusion#632] [Q&A] [answered]
I want to fix the hand mistake generated by AI.
After using "selection tool" to select the hand of the character, I went to live painting and click "start". The result is still the redraw of the whole image instead of "just the hand".
Could anyone tell me how do I apply my selection area?

**Comment by Acly** [comment, marked as answer]
There was an issue about this in a much earlier version few months ago: #206 
Apart from creating a selection nothing more should be required, it's still working for me.

**Reply by brightcxy** [reply]
Seems my plugin version is not correct. I can't see the version number displayed on the bottom of the settings page.
I tried re-import the latest zip file but still not working, no version number displayed.

**Reply by Acly** [reply]
That seems to be a common issue, probably some file is in use and prevents replacement. Try to delete the plugin folder, or disable the plugin in Krita options, then restart and do the import.
For install location see https://github.com/Acly/krita-ai-diffusion/wiki/Common-Issues#where-are-plugin-and-user-files-located

**Reply by brightcxy** [reply]
After replacing the whole folder, the latest version number displayed. Then after the automatic setup and install, problem solved. Thank you!
